tests/pytorch/fused_attn/test_fused_attn.py

```diff
@@ -1527,8 +1528,8 @@ def test_dpa_fp8_vs_f16(dtype, model, qkv_layout, fp8_dpa_bwd, is_training):
     os.environ["NVTE_FP8_DPA_BWD"] = "1" if fp8_dpa_bwd else "0"
     os.environ["NVTE_ALLOW_NONDETERMINISTIC_ALGO"] = "1"
 
-    global _attention_backends
-    if not is_training:
+    global _attention_backends, _use_flash_attn_3
```

Comment:
I don't think this is necessary since we never assign to `_attention_backends` or `_use_flash_attn_3`:
```suggestion
```

Reply:
Removed.